core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionPolicyManager.java

```diff
@@ -46,8 +50,17 @@
  *
  * Created by gupele on 6/29/2015.
  */
-public final class TransmissionPolicyManager implements Stoppable {
-
+public final class TransmissionPolicyManager implements Stoppable, TransmissionHandlerObserver {
+
+	private int INSTANT_RETRY_AMOUNT = 3; // Should always be set by the creator of this class
```

Comment:
this is using the java naming convention for constants. For member variables, `camelCase` is preferred, but I'm not going to block the PR for this :)

Reply:
Fair enough.